dotcom-rendering/src/web/layouts/FrontLayout.tsx

```diff
@@ -258,22 +259,48 @@ export const FrontLayout = ({ front, NAV }: Props) => {
 
 					const ophanName = ophanComponentId(collection.displayName);
 					const ophanComponentLink = `container-${index} | ${ophanName}`;
+					const COTTON_CAPITAL_THRASHERS = [
+						'https://content.guardianapis.com/atom/interactive/interactives/2022/10/tr/default-fronts-default',
+						'https://content.guardianapis.com/atom/interactive/interactives/2022/10/tr/david-olusoga-front-default',
+						'https://content.guardianapis.com/atom/interactive/interactives/2022/10/tr/cassandra-gooptar-front-default',
+						'https://content.guardianapis.com/atom/interactive/interactives/2022/10/tr/gary-younge-front-default',
+						'https://content.guardianapis.com/atom/interactive/interactives/2022/10/tr/deneen-l-brown-front-default',
+						'https://content.guardianapis.com/atom/interactive/interactives/2022/10/tr/the-enslaved-front-default',
+						'https://content.guardianapis.com/atom/interactive/interactives/2022/10/tr/olivette-otele-front-default',
+						'https://content.guardianapis.com/atom/interactive/interactives/2022/10/tr/interactives-front--globe',
+						'https://content.guardianapis.com/atom/interactive/interactives/2022/10/tr/michael-taylor-front-default',
+						'https://content.guardianapis.com/atom/interactive/interactives/2022/10/tr/lanre-bakare-front-default',
+						'https://content.guardianapis.com/atom/interactive/interactives/2022/10/tr/hidden-figures-front-default',
+						'https://content.guardianapis.com/atom/interactive/interactives/2022/10/tr/johny-pitts-photo-essay-front-default',
+					];
 
 					if (collection.collectionType === 'fixed/thrasher') {
 						return (
 							<>
-								<Section
-									fullWidth={true}
-									padSides={false}
-									padBottom={true}
-									showTopBorder={false}
-									showSideBorders={true}
-									ophanComponentLink={ophanComponentLink}
-									ophanComponentName={ophanName}
-									containerName={collection.collectionType}
-								>
+								{!!trail.embedUri &&
+								COTTON_CAPITAL_THRASHERS.includes(
+									trail.embedUri,
+								) ? (
 									<Snap snapData={trail.snapData} />
-								</Section>
+								) : (
+									<SnapWrapper snapData={trail.snapData}>
```

Comment:
Could we name `SnapWrapper` something like `SnapCssSandbox`? I think this would help someone immediately understand the structure :)